sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md

```diff
@@ -3,12 +3,40 @@
 ## 1.0.0b4 (Unreleased)
 
 ### Features Added
+- Added support for the Analyze Batch Documents API:
```

Comment:
Do you plan to add tests/samples for some of these new features?

Reply:
Yes, tests and samples for new features will be in a separate PR. Additionally, we need some customization for some new features, working on it now.